src/api/freeFax/freeFax.v6.service.js

```diff
@@ -19,6 +19,10 @@ angular.module('ovh-api-services').service('OvhApiFreeFaxV6', ($resource, $cache
       isArray: true,
       cache: OvhApiFreeFax.cache,
     },
+    getServiceInfos: {
```

Comment:
Is the `PUT` not needed (https://api.ovh.com/console/#/freefax/%7BserviceName%7D/serviceInfos#PUT)?